src/main/java/org/broadinstitute/dsm/db/EditParticipantMessage.java

```diff
@@ -226,4 +191,25 @@ public static void updateMessageStatusById(@NonNull int messageId, @NonNull Stri
         }
     }
 
+    public static void deleteMessage(int userId) {
```

Comment:
if that method is only used in the unit test, I would prefer moving it to the test class